docs/versioned-plugins/include/version-list-intro.asciidoc

```diff
@@ -0,0 +1,13 @@
+[id="{type}-{plugin}-index"]
+
+== Versioned {plugin} {type} plugin docs
+++++
+<titleabbrev>{plugin}</titleabbrev>
+++++
+
+This page lists all available versions of the documentation for this plugin. 
+To see which version of the plugin you have installed, run `bin/logstash-plugin
+list --verbose`. 
+
+NOTE: Versioned plugin documentation is not available for plugins released prior
```

Comment:
I think the actual restriction here is we can only do this easily for plugin versions that have the separated index.asciidoc (instead of the inline documentation), no? That said, I agree we can put a "starting marker" in the Logstash 6.0 release

Reply:
Yup... 6.0 is just a placeholder for whatever we want to say based on how far back we want to go. We were still making manual hacks to the generated asciidoc during the 5.x releases, so we probably don't want to go back too far or we'll have a bunch of broken asciidoc stuff to fix.  We could be a bit wishy washy and say something like, "Versioned plugin documentation was added to the Elastic docs during the Logstash 6.0 release. Therefore the documentation for plugins released prior to 6.0 might not be available here."